kafka-ui-react-app/src/components/common/PageLoader/PageLoader.tsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import * as SP from 'components/common/Spinner/Spinner.styled';
 
 import * as S from './PageLoader.styled';
 
 const PageLoader: React.FC = () => (
   <S.Wrapper>
-    <S.Spinner role="progressbar" />
+    <SP.Spinner
```

Comment:
We use S namespace for all styled components related to current component. 
```suggestion
    <Spinner
```